Gruntfile.js

```diff
@@ -90,7 +90,7 @@ module.exports = function(grunt) {
         options: {
           // The src/vendor paths are needed to find the CF components' files.
           // Feel free to add additional paths to the array passed to `concat`.
-          paths: grunt.file.expand('src/vendor/*').concat([])
+          paths: '<%= loc.src %>/vendor/'
```

Comment:
Adding the vendor path means that you don't need the full path when importing with Less here https://github.com/cfpb/design-manual/pull/341/files#diff-32dca36d08c77c57a0a184ef881257eaR10.

Good catch on the comment, will remove